app/renderer/components/WrongFolder/WrongFolder.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Button } from 'antd';
+import { ipcRenderer } from 'electron';
+import WrongFolderStyles from './WrongFolderStyles.css';
+
+export default class WrongFolder extends Component {
+  // This tells the main thread to move this to applications folder which will cause the app to close and restart
+  moveToApplicationsFolder () {
+    ipcRenderer.send('appium-move-to-applications-folder');
+  }
+
+  render () {
+    return <div className={WrongFolderStyles['wrong-folder']}>
+      <div>
+        <div>Appium Desktop should be run from the Applications folder</div>
```

Comment:
Do we show the same message on Linux/Windows?

Reply:
No the function only applies to Mac.

Reply:
do we exit the app if this dialog is canceled?

Reply:
Yeah. Do you think we should give them the option to proceed without moving the folder?

Reply:
definitely not ;)